### PR TITLE
Upgrade to React16

### PR DESCRIPTION
**Pull request by adamginzberg**
PTAL: @vaskevich @fortes

### CODE REVIEW

**Review by fortes — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/missive/emoji-mart-lite",
   "dependencies": {},
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0"
+    "react": "^0.14.0 || ^15.0.0-0 || ^16.0.0"
```
Comment (fortes) [inline package.json]: Let's just shorten this to React latest? `16.1` or whatever it is

**Review by oleg-codaio — APPROVED** [review]
That's it, lol? LGTM.

**Comment by fortes** [comment]
Still gotta version bump and then `npm publish` though.

**Comment by vaskevich** [comment]
@fortes you're an owner on this package AFAIR - could you publish? I'm OoO.

**Comment by fortes** [comment]
Great, merging and will publish.
